data-collections-exercises.md

```diff
@@ -9,95 +9,95 @@ planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
 Access the second value in `planeteers`.
 
 ```rb
-# Your answer here
+planeteers[1]
 ```
 
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+planeteers.push('heart')
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+planeteers.delete('captin planet')
```

Comment:
you used a delete a method, the goal is to experience this cool feature that ruby where you can do subtraction !! `planeteers. - ["Captain Planet"]`